src/services/session.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const protocolCallbacks = require('@services/protocolCallbacks/')
+const sessionAttendanceQueries = require('@database/storage/sessionAttendance/queries')
+const userQueries = require('@database/storage/user/queries')
+const bapQueries = require('@database/storage/bap/queries')
+const crypto = require('crypto')
+
+exports.session = async (requestBody) => {
+	try {
+		const sessionId = requestBody.sessionId
+		const sessionAttendance = await sessionAttendanceQueries.findBySessionId(sessionId)
+		if (!sessionAttendance) {
+			return console.log('SessionAttendance Not Found')
+		}
+		const userIds = sessionAttendance.map((sessionAttendee) => sessionAttendee.userId)
```

Comment:
"sessionAttendee" can be a bit confusing here since we already have a sessionAttendeeId in the model. And sessionAttendee is a mentoring service side concept. Just "attendance" would be better than "sessionAttendee" since that doesn't change the idea of what that object represent (which is an attendance, and not attendee).